core/app/models/spree/variant.rb

```diff
@@ -194,7 +194,7 @@ def is_backorderable?
     #
     # @return [String] a sentence-ified string of option values.
     def options_text
-      values = option_values.includes(:option_type).sort_by do |option_value|
```

Comment:
Maybe we can add a comment on top of the method to document that it assumes there's a preload of the option_type in the context where it's used. Otherwise, suggest people to do it.

Reply:
Also, maybe we can do it programmatically using something like  `option_values.association(:option_type).loaded?`

Reply:
Hm, many of the instance methods we have access related objects pretty willy-nilly. If I add that comment here, that suggests we'd have to do that in lots of other places as well. 

I don't really want to do that, nor the check whether the association is loaded, because IMO optimizing for n+1s is something that should happen at the controller level and not in model instances.